content-src/components/Spotlight/Spotlight.js

```diff
@@ -48,10 +48,15 @@ const Spotlight = React.createClass({
         return !!(site.images && site.images[0] && site.images[0].url);
       })
       .slice(0, this.props.length);
+    const blankSites = [];
+    for (let i = 0; i < (this.props.length - sites.length); i++) {
+      blankSites.push(<li className="spotlight-item spotlight-placeholder" />);
+    }
     return (<section className="spotlight">
       <h3 className="section-title">Spotlight</h3>
       <ul>
         {sites.map(site => <SpotlightItem key={site.url} {...site} />)}
+        {blankSites}
```

Comment:
Ah, i wonder if this is what caused the following errors in the DevTools console:

> "Warning: Each child in an array or iterator should have a unique "key" prop. Check the render method of `Spotlight`. See https://fb.me/react-warning-keys for more information."
> &mdash; via Spotlight.js#render():L89

I'm not sure exactly what line the error is on in the `render()` method, because it looks like the React code is getting transpiled down into this:

```js
      React.createElement(
        "ul",
        null,
        sites.map(site => React.createElement(SpotlightItem, _extends({ key: site.url }, site))),
        blankSites
      )
```